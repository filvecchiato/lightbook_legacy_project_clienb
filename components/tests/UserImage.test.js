import React from 'react';
import renderer from 'react-test-renderer';
import UserImage from '../UserImage';

test('renders correctly', () => {
  const tree = renderer.create(<UserImage image={{ url: 'test' }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
