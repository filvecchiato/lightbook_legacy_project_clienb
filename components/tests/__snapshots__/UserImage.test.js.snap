// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "aspectRatio": 1.7,
      "resizeMode": "cover",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "test",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
</View>
`;
